api-gateway/src/main/resources/static/scripts/bill-history/bill-history.controller.js

```diff
@@ -2,12 +2,19 @@
 
 
 angular.module('billHistory')
-    .controller('BillHistoryController', ['$http', '$scope', function ($http, $scope) {
+    .controller('BillHistoryController', ['$http', '$scope', '$stateParams', function ($http, $scope, $stateParams) {
         var self = this;
 
+        let requestUrl = "api/gateway/bills/";
+
+        if($stateParams.filterId != null ){
+            requestUrl += $stateParams.filterType + "/" + $stateParams.filterId;
+        }
+
+        console.log(requestUrl);
```

Comment:
Needs to be removed or at least commented out

Reply:
Removed the two console.log